installer/templates/phx_live/templates/layout/root.html.leex

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <%%= live_title_tag assigns[:page_title] || <%= app_module %>, suffix: " · Phoenix Framework" %>
+    <%%= live_title_tag assigns[:page_title] || "<%= app_module %>", suffix: " · Phoenix Framework" %>
```

Comment:
I was thinking that maybe, for simplicity, we just do this:

```suggestion
    <%%= live_title_tag assigns[:page_title] || "<%= app_module %> · Phoenix Framework" %>
```

And let people explore the rest if they want to?